cluster-autoscaler/main.go

```diff
@@ -257,7 +257,9 @@ func createAutoscalingOptions() config.AutoscalingOptions {
 			ScaleDownGpuUtilizationThreshold: *scaleDownGpuUtilizationThreshold,
 			ScaleDownUnneededTime:            *scaleDownUnneededTime,
 			ScaleDownUnreadyTime:             *scaleDownUnreadyTime,
-			MaxNodeProvisionTime:             *maxNodeProvisionTime,
+			// per nodegroup setting, defaulted to global setting
```

Comment:
nit: Is this comment needed? Or at least - is it needed here? It's just as true for every other setting in this struct

Reply:
Thank you for the comment. I don't think this comment is needed anymore (leftover from initial implementation of this PR). I have removed it.